test/pieces.js

```diff
@@ -568,8 +572,15 @@ describe('Pieces', function() {
               {
                 metaType: 'widget',
                 type: '@apostrophecms/rich-text',
-                id: cuid(),
+                _id: (i === 1) ? widgetId : null,
```

Comment:
Test consequences of an explicit widget id that duplicates another, but only in the first document in the set.